packages/react-data-grid/src/Canvas.js

```diff
@@ -414,9 +414,6 @@ class Canvas extends React.PureComponent {
           onCellRangeSelectionCompleted={this.props.onCellRangeSelectionCompleted}
           scrollLeft={this._scroll.scrollLeft}
           scrollTop={this._scroll.scrollTop}
-          prevScrollLeft={this.props.prevScrollLeft}
```

Comment:
This props do not seem to be used anywhere

Reply:
I think they can be safely removed.